synapseclient/models/protocols/schema_organization_protocol.py

```diff
@@ -0,0 +1,292 @@
+"""Protocols for the specific methods of this class that have synchronous counterparts
+generated at runtime."""
+
+from typing import TYPE_CHECKING, Any, Mapping, Optional, Protocol, Sequence
+
+if TYPE_CHECKING:
+    from synapseclient import Synapse
+    from synapseclient.models.mixins.json_schema import JSONSchemaVersionInfo
+    from synapseclient.models.schema_organization import JSONSchema
+
+
+class SchemaOrganizationProtocol(Protocol):
+    """
+    The protocol for methods that are asynchronous but also
+    have a synchronous counterpart that may also be called.
+    """
+
+    def get(self, synapse_client: Optional["Synapse"] = None) -> None:
+        """
+        Gets the metadata from Synapse for this organization
+
+        Arguments:
+            synapse_client: If not passed in and caching was not disabled by
+                `Synapse.allow_client_caching(False)` this will use the last created
+                instance from the Synapse class constructor
+
+        Example:
+            from synapseclient.models import SchemaOrganization
+            from synapseclient import Synapse
+
+            syn = Synapse()
+            syn.login()
+
+            org = SchemaOrganization("my.org.name")
+            org.get()
```

Comment:
This formatting is going to break in mkdocs. It'll need to be something like:

```suggestion
        Example: A short description of the example
        Additional information or a `&nbsp;` string instead to add a whitespace character
        
        ```python
        from synapseclient.models import SchemaOrganization
        from synapseclient import Synapse

        syn = Synapse()
        syn.login()

        org = SchemaOrganization("my.org.name")
        org.get()
        ```

```